scripts/install2

```diff
@@ -11,13 +11,18 @@ BOARD_REVISION="?"
 CJDNS_BUILD_CMD="sudo Seccomp_NO=1 NO_NEON=1 ./do"
 BOARD_HARDWARE=$(cat /proc/cpuinfo  | grep Hardware | awk '{print $3}' | head -n 1)
 
+ALLOW_HOSTAP=false #Flag for prompt of hostap
```

Comment:
`SUPPORT_HOSTAP`? The comment for the flag should be whether the board _supports_ hostapd, the prompt is just the effect of the flag being true.